test/unit/math/rev/prob/binomial_lpmf_test.cpp

```diff
@@ -14,8 +14,8 @@ TEST(ProbBinomial, n_equals_N) {
 TEST(ProbBinomial, n_equals_N_vec) {
   using stan::math::var;
   var theta = 1.0;
-  std::vector<int> n = { 2, 3 };
-  std::vector<int> N = { 2, 3 };
+  std::vector<int> n = {2, 3};
```

Comment:
There should be new tests for the behavior that failed before and is fixed now.  This PR just changed the tests' formatting.

There should also be a full set of boundary condition tests for 

```
binomial_lpmf({0, N}, N, {0, 0.5, 1})  for N in {0, 1, 2}
```

That should test both scalars and vectors for the `N = 1` cases.

Reply:
I think Github must have showed you the diff only for the Jenkins reformatting.

Your test is easy to add and makes sense so I will add it.